kong/plugins/session/session.lua

```diff
@@ -62,21 +63,24 @@ function _M.retrieve_session_data(s)
     return nil, nil
   end
 
-  return s.data[1], s.data[2]
+  return s.data[1], s.data[2], s.data[3]
 end
 
 
 --- Store the session data for usage in kong plugins
 -- @param s - the session
 -- @param consumer - the consumer id
 -- @param credential - the credential id or potentially just the consumer id
-function _M.store_session_data(s, consumer_id, credential_id)
+-- @param groups - table of authenticated_groups e.g. ["1" = "group1", "group1"]
```

Comment:
is the format of the table example here correct?

Reply:
updated to remove hash and be proper lua table `{}`